gamemode/modules/util/cl_radial_menu.lua

```diff
@@ -0,0 +1,155 @@
+surface.CreateFont( "RVR_RadialMenu_Title", {
+    font = "Bungee Regular",
+    size = 30,
```

Comment:
Hard coded font size is risky, we've used `ScrH() * 0.0whatever` elsewhere in the project